tests/test_wrappers.py

```diff
@@ -128,124 +268,76 @@ def test_decode_byte_xml():
                             }
                         },
                         "Relationships": {},
-                        "SalesRankings": {
```

Comment:
These types of entries just bloated the test, in my opinion, so I removed these sections.